qurt_lib.cmake

```diff
@@ -98,23 +98,20 @@ function (QURT_LIB)
 			)
 
 		add_dependencies(${QURT_LIB_LIB_NAME} generate_${QURT_LIB_IDL_NAME}_stubs)
-
 	endif()
 
 	add_library(${QURT_LIB_IDL_NAME}_skel MODULE
 		${QURT_LIB_IDL_NAME}_skel.c
 		)
 
-	if (NOT "${QURT_LIB_SOURCES}" STREQUAL "")
-		target_link_libraries(${QURT_LIB_IDL_NAME}_skel
-			${QURT_LIB_LIB_NAME}
-			)
-	else()
-		target_link_libraries(${QURT_LIB_IDL_NAME}_skel
-			${QURT_LIB_LINK_LIBS}
```

Comment:
This seems to completely eliminate the ability to add other library deps via QURT_LIB_LINK_LIBS. Something needs to change here. Either QURT_LIB_LINK_LIBS should be entirely removed, or it needs to use used somewhere.